knowledge-base/actions/deborah-digges/new-pull-request-comment-action/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Welcome bot on new pull requests'
+github-token:
+  action-input:
+    input: access-token
+    is-default: false
+  permissions:
```

Comment:
I think there is also `pull-request: read` permission `to check if it is first pull request`, view [here](https://github.com/Deborah-Digges/new-pull-request-comment-action/blob/224c179a9e23f65ec50ff3240b8716369dc415d7/src/index.js#L9).